packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -281,6 +460,20 @@ export default class StatsAnalyzer extends EventsScope {
     this.peerConnection.shareTransceiver.receiver.getStats().then((res) => {
       this.filterAndParseGetStatsResults(res, STATS.SHARE_CORRELATE, false);
     });
+
+    // updates the current direction of media
+    this.statsResults[STATS.AUDIO_CORRELATE].direction = this.peerConnection.audioTransceiver.currentDirection;
+    this.statsResults[STATS.VIDEO_CORRELATE].direction = this.peerConnection.videoTransceiver.currentDirection;
+    this.statsResults[STATS.SHARE_CORRELATE].direction = this.peerConnection.shareTransceiver.currentDirection;
+
+    // Process Stats results every 5 seconds
+
+    this.compareLastStatsResult();
+
+    // Save the last results to compare with the current
+    // DO Deep copy, for some reason it takes the reference all the time rather then old value set
+    this.lastStatsResults = JSON.parse(JSON.stringify(this.statsResults));
```

Comment:
Can you use the `cloneDeep` lodash function instead as it is used elsewhere in the sdk?

```js
import {cloneDeed} from 'lodash';
```

Reply:
will give a try i think i tried clone before and it was taking it as a reference